test/controllers/TransferSummaryControllerSpec.scala

```diff
@@ -0,0 +1,132 @@
+package controllers
+
+import java.util.UUID
+
+import com.github.tomakehurst.wiremock.WireMockServer
+import com.github.tomakehurst.wiremock.client.WireMock.{okJson, post, urlEqualTo}
+import configuration.GraphQLConfiguration
+import errors.AuthorisationException
+import graphql.codegen.GetConsignment.{getConsignment => gc}
+import io.circe.Printer
+import io.circe.generic.auto._
+import io.circe.syntax._
+import org.scalatest.Matchers._
+import org.scalatest.concurrent.ScalaFutures._
+import play.api.Play.materializer
+import play.api.i18n.Langs
+import play.api.test.CSRFTokenHelper._
+import play.api.test.FakeRequest
+import play.api.test.Helpers.{GET, contentAsString, contentType, redirectLocation, status => playStatus, _}
+import uk.gov.nationalarchives.tdr.GraphQLClient
+import uk.gov.nationalarchives.tdr.GraphQLClient.Extensions
+import util.{EnglishLang, FrontEndTestHelper}
+
+import scala.concurrent.ExecutionContext
+
+class TransferSummaryControllerSpec extends FrontEndTestHelper {
+  implicit val ec: ExecutionContext = ExecutionContext.global
+
+  val wiremockServer = new WireMockServer(9006)
+
+  override def beforeEach(): Unit = {
+    wiremockServer.start()
+  }
+
+  override def afterEach(): Unit = {
+    wiremockServer.resetAll()
+    wiremockServer.stop()
+  }
+
+  val langs: Langs = new EnglishLang
+
+  "TransferSummaryController GET" should {
+
+    "render the transfer summary page with an authenticated user" in {
+      val consignmentId = UUID.fromString("c2efd3e6-6664-4582-8c28-dcf891f60e68")
```

Comment:
The same consigmentId is used in multiple tests, so Id be tempted to move this to a global val to avoid repeated code.

Reply:
Good point! Thanks.